pkg/executor/executortype/container/common.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2020 The Fission Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
+
+import (
+	"context"
+	"strconv"
+
+	multierror "github.com/hashicorp/go-multierror"
+	"go.uber.org/zap"
+	apiv1 "k8s.io/api/core/v1"
+	k8s_err "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	fv1 "github.com/fission/fission/pkg/apis/core/v1"
+)
+
+// getResources gets the resources(CPU, memory) set for the function
+func (cn *Container) getResources(fn *fv1.Function) apiv1.ResourceRequirements {
```

Comment:
nitpick - wonder if we can modify the similar method here [func (deploy *NewDeploy) getResources(env *fv1.Environment, fn *fv1.Function) apiv1.ResourceRequirements {](https://github.com/fission/fission/blob/04350fc0d7c18baa23158388fc22d989cd61aa30/pkg/executor/executortype/newdeploy/newdeploy.go#L337-L338) to reuse common logic. We can also refactor later, not going to hold this PR for just this

Reply:
Hi Vishal, we will fix it in another cycle. There are lot of places where we need to combine code across executors.